cloudbuild.yaml

```diff
@@ -268,13 +270,50 @@ steps:
 - name: 'e2e-runner'
   args:
     - ''
+    - 'generic'
     - 'e2e-test-cluster'
+    - 'us-west1-c'
     - "${_REGISTRY}"
   id: e2e-stable
   waitFor:
     - push-images
     - build-e2e
 
+#
+# Run the e2e tests on GKE Autopilot with FeatureGates inverted compared to Stable
+#
+# Keep in sync with (the inverse of) pkg/util/runtime/features.go:featureDefaults
+
+- name: 'e2e-runner'
```

Comment:
Just to leave a note here, this will be refactored a little bit if we have consensus on the CI change proposed in the design of multiple version support. But definitely looks good for now.

Reply:
Yeah, this was meant as an incremental addition for sure.